src/test/java/reactor/core/publisher/tck/FluxAndProcessorTests.java

```diff
@@ -67,7 +67,7 @@
 		                                                                   .doOnNext(this::monitorThreadUse))
 		                                      .doOnNext(array -> cumulatedJoin.getAndIncrement())
 		                                      .subscribeWith(TopicProcessor.create("fluxion-raw-join", bufferSize))
-		                                      .as(Flux::from)
+		                                      .transform(Flux::from)
```

Comment:
It tests the `as` operator, they should have some test coverage so an unwanted change can be captured.